pkg/sessionctx/stmtctx/stmtctx.go

```diff
@@ -1458,6 +1458,8 @@ type UsedStatsInfoForTable struct {
 	ModifyCount           int64
 	ColumnStatsLoadStatus map[int64]string
 	IndexStatsLoadStatus  map[int64]string
+	StatsTbl              interface{}
+	ColAndIdxStatus       interface{}
```

Comment:
Could you add more comments about these 2 fields? Add can we use a specific type instead of interface{} here?

Reply:
It will cause cyclic importing